Public/Src/Cache/ContentStore/Distributed/Stores/DistributedContentCopier.cs

```diff
@@ -281,6 +289,18 @@ public Task<BoolResult> RequestCopyFileAsync(OperationContext context, ContentHa
             {
                 var location = hashInfo.Locations[replicaIndex];
 
+                //Currently everytime we increment attemptCount's value, we go through every location in hashInfo and try to copy.

```

Comment:
very nit: we usually separate start of the comment with the text with a space. I.e. "// Currently ..."